### ISSUE TITLE
Idea for Evil/Off-Hand Faiths - Nightmare Druids/Void Worshipping Nature Faith

### ISSUE THREAD

**Issue by LoneWolfGamer56** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#1179]
**Describe your suggestion in full detail below:**

Hello GoA Team! This may be an idea you've heard before or remember being suggested in the CK2 mod adaptation; specifically, I once suggested a Nightmare Druid subclass and faith offshoot of the Shath'yar Void religion surrounding corrupted world trees and an emerging event line bringing them to a forefront, similar to the old Scarlet Light religion.

With the addition of gods and the ideas of pantheons in CK3, this could be fleshed out further, perhaps a Nightmare/Void World Tree focused faith starting with one specifically for Night Elves with different caveats of tenets. How any of this would be implemented, I leave for you to think about. 

TLDR; Old Gods or Void based evil/neutral faith(s) for evil Druid players, event lines and decisions for emerging religion should a World Tree be corrupted by a Black Empire player or AI.
